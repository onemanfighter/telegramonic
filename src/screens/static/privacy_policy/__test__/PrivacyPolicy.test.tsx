import { render } from '@testing-library/react';
import PrivacyPolicy from '../PrivacyPolicy';

describe('PrivacyPolicy screen', () => {
  it('should render correctly', () => {
    const { container } = render(<PrivacyPolicy />);

    expect(container).toMatchSnapshot();
  });
});
