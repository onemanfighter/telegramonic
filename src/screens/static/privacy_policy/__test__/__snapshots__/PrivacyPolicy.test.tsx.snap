// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivacyPolicy screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Privacy Policy
    </h1>
  </div>
</div>
`;
